src/ui.h

```diff
@@ -15,7 +17,7 @@
 // Special ids for windows size and position
 #define SIZE_FROM_IMAGE  0
 #define SIZE_FROM_PARENT 1
-#define POS_FROM_PARENT  0xffffffff
+#define POS_FROM_PARENT  SSIZE_MAX
```

Comment:
I don't see a reason to change this. What the purpose?

Reply:
`0xffffffff` may be too large for an `ssize_t`, and some functions (`ui_get_x` and `ui_get_y`) return this value as an `ssize_t`. I see that `POS_FROM_PARENT` is a special return value: normal positions can be returned, so can this very large value, which has a special meaning. This way, `SSIZE_MAX` is for sure the greatest value that can fit in an `ssize_t`.